profiles/seccomp/seccomp_default.go

```diff
@@ -1297,7 +1297,15 @@ func DefaultProfile(rs *specs.Spec) *types.Seccomp {
 		{
 			Name:   "setsockopt",
 			Action: types.ActAllow,
-			Args:   []*types.Arg{},
+			Args: []*types.Arg{
+				{
+					// disallow IPT_SO_SET_REPLACE, IP6T_SO_SET_REPLACE, ARPT_SO_SET_REPLACE
+					Index:    2,
```

Comment:
We should only disallow `IPT_SO_SET_REPLACE` and `ARPT_SO_SET_REPLACE` when arg 1 (level) is `SOL_IP` (0) and `IP6T_SO_SET_REPLACE` when arg 1 is `SOL_IPV6` (41), otherwise we might be blocking valid socket options on different address families.